atlas-db-controller/databaseSchemaResource.go

```diff
@@ -146,7 +148,15 @@ func (c *Controller) syncSchema(key string) error {
 
 	ver, dirt, err := mgrt.Version()
 	if err != nil {
-		if err == migrate.ErrNilVersion {
+		errString := err.Error()
+		if strings.Contains(errString, BAD_CONN) || strings.Contains(errString, CONN_CLOSED) {
+			delete(dbDriverMap, db_key)
+			schemaStatusMsg = fmt.Sprintf("database connection errorred: %s", err)
+			c.updateDatabaseSchemaStatus(key, schema, StateError, schemaStatusMsg)
+			err = fmt.Errorf(schemaStatusMsg)
```

Comment:
Could this be just a `warning` instead of `error`  ?

Reply:
`fmt.Warning` || `fmt.Warningf` does not exists. Moreover it is just the string formatting

Reply:
I was not taking about the `formatting`. Since we are returning err and doing `runtime.HandleError(err)`

Reply:
if we don't return here.. it will raise a false event `Successfully synced schema` from line 196.